test/files/jvm/varargs-separate-bytecode/Props_1.flags

```diff
@@ -0,0 +1 @@
+-sourcepath test/files/jvm/varargs-separate-bytecode/
```

Comment:
You're right, it's being used in presentation compiler tests only. What I need is to make sure that `AbstractProps` is not compiled together with `Props.scala`. Is there a way to compile first one set of file, and afterwards another set of files?